### PR TITLE
feat(content): added bot flow for self-assessment tool created by provincial authorities

### PR DESCRIPTION
**Pull request by vanessajones**
Added bot flow for provinces with self-assessment tools.

Workflow:
- If province is Ontario or Alberta, the user is displayed with the provincial self-assessment tool, and then asked if they want to continue.
- If they say no (don't want to continue), the bot thanks the user and flow ends there. 
- If they say yes (wants to continue), the user will be taken to the main assessment.

Screen Shot 2020-03-16 at 7 32 10 PM

### CODE REVIEW

File: src/regions/ca/i18n/steps.fr.ts [hunk 1]
```diff
@@ -78,5 +78,25 @@ export default {
...
+  Si vous pensez que vous avez les symptômes de la maladie à coronavirus (COVID-19) ou si vous avez été en contact étroit avec une personne qui en est atteinte, veuillez utiliser cet outil d’évaluation créer par le gouvernement d'Ontario afin de déterminer comment obtenir des soins supplémentaires.
+    
+  Passez l'évaluation développée par le gouvernement de l'Ontario [ici.](https://www.ontario.ca/page/2019-novel-coronavirus-covid-19-self-assessment)
+  `,
+  'assessment-continue': `
+  Je peux également vous fournir des informations pertinentes et récentes sur COVID-19. Voulez-vous continuer ?`,
+  outro2: `
+  Très bien ! Merci d'avoir visité ce site. 
+  
+  N'oubliez pas que vous pouvez toujours revenir pour obtenir votre dossier d'information personnalisé sur COVID-19.
```
Comment (hollyorlando) [inline src/regions/ca/i18n/steps.fr.ts]:
```suggestion
  N'oubliez pas que vous pouvez toujours revenir pour obtenir votre dossier d'information personnalisé sur le COVID-19.
```

File: src/regions/ca/i18n/steps.fr.ts [hunk 2]
```diff
@@ -78,5 +78,25 @@ export default {
...
   Les questions sont terminées ! Votre trousse d'information personnelle est prête.
+  `,
+  'assessment-AB': `
+  Les symptômes de la COVID-19 vous inquiètent ?
+  
+  Utilisez cet outil d'auto-évaluation développé par le gouvernement de l'Alberta pour vous aider à déterminer si vous devez être testé pour le COVID-19. Vous pouvez effectuer cette évaluation pour vous-même ou pour le compte d'une autre personne, si elle n'est pas en mesure de le faire.
+  
+  Passez l'évaluation développée par le gouvernement de l'Ontario [ici (en anglais).](https://myhealth.alberta.ca/journey/covid-19/Pages/COVID-Self-Assessment.aspx)`,
+  'assessment-ON': `
+  Les symptômes de la COVID-19 vous inquiètent ?
```
Comment (hollyorlando) [inline src/regions/ca/i18n/steps.fr.ts]:
```suggestion
  Les symptômes du COVID-19 vous inquiètent ?
```

File: src/regions/ca/i18n/steps.fr.ts [hunk 3]
```diff
@@ -78,5 +78,25 @@ export default {
...
+  
+  Si vous pensez que vous avez les symptômes de la maladie à coronavirus (COVID-19) ou si vous avez été en contact étroit avec une personne qui en est atteinte, veuillez utiliser cet outil d’évaluation créer par le gouvernement d'Ontario afin de déterminer comment obtenir des soins supplémentaires.
+    
+  Passez l'évaluation développée par le gouvernement de l'Ontario [ici.](https://www.ontario.ca/page/2019-novel-coronavirus-covid-19-self-assessment)
+  `,
+  'assessment-continue': `
+  Je peux également vous fournir des informations pertinentes et récentes sur COVID-19. Voulez-vous continuer ?`,
```
Comment (hollyorlando) [inline src/regions/ca/i18n/steps.fr.ts]:
```suggestion
  Je peux également vous fournir des informations pertinentes et récentes sur le COVID-19. Voulez-vous continuer ?`,
```

File: src/regions/ca/steps.json [hunk 4]
```diff
@@ -328,5 +328,40 @@
...
+        "label": "no",
+        "value": false,
+        "trigger": "outro2"
+      }
+    ]
+  },
+  {
+    "id": "outro2",
+    "message": "outro2",
+    "trigger": ""
```
Comment (vicrep) [inline src/regions/ca/steps.json]: Btw, you shouldn't have empty triggers, but rather mark those steps with `end: true` -- just make sure the rules / class system can handle it :)

File: src/regions/ca/i18n/steps.fr.ts (hunk 1 above)
Comment (vicrep) [reply src/regions/ca/i18n/steps.fr.ts]: Technically, it's `la` (for `la maladie coronavirus`) -- it sounds weird / off I know

File: src/regions/ca/steps.json (hunk 4 above)
Comment (vanessajones) [reply src/regions/ca/steps.json]: @vicrep do you have an example? trying to find one in the docs but cannot

Comment (vicrep) [reply src/regions/ca/steps.json]: Look at user step here: https://lucasbassetti.com.br/react-simple-chatbot/#/docs/steps it's not specified in docs but same can be done for a message step

File: src/regions/ca/steps.json [hunk 5]
```diff
@@ -362,6 +362,6 @@
   {
     "id": "outro2",
     "message": "outro2",
-    "trigger": ""
+    "end": true
```
Comment (vanessajones) [inline src/regions/ca/steps.json]: @vicrep I changed it to this - seems to work (it ends the flow) but not sure if that's what you meant?

File: src/regions/ca/steps.json (hunk 4 above)
Comment (vanessajones) [reply src/regions/ca/steps.json]: Thanks!

Comment (vanessajones) [reply src/regions/ca/steps.json]: Thanks!

**Comment by vanessajones** [comment]
BTW the government of Canada uses "le COVID-19 or du COVID-19"

**Pull request merged**